crux-llvm/test-data/golden/golden/invoke-test.pre-clang16.z3.good

```diff
@@ -5,7 +5,7 @@
 [Crux]     No implementation or override found for pointer
 [Crux]     The given pointer could not be resolved to a callable function
 [Crux]     No implementation or override found for pointer
-[Crux]     Attempting to load callable function with type: i64({ }*, [3 x i64]*, i64, i8**)
+[Crux]     Attempting to load callable function:
```

Comment:
Note to reviewers: I removed an llvm-pretty type from the LLVM syntax former for loading function handles from pointers, because it was only used to generate this diagnostic message and it seems unnecessary, redundant, and mildly effortful to add a parser for LLVM types to `crucible-llvm-syntax`. This makes this error message less informative than it formerly was. With #1016, we could at least pretty-print the Crucible types involved, but for now this seems like an acceptable trade-off. The LLVM type can always be recovered by looking at the `llvm-dis` output.